### ISSUE TITLE
Removing the admin from the system contract entirely. 

### ISSUE THREAD

**Issue by piux2** [gnolang/gno#2657]
For better security and trust of the chain, we should consider removing the admin from the system contract entirely. No one should be able to add a contract under /r/system unless it is voted on by the gov DAO or through a chain upgrade that involves consent from the validators.

_Originally posted by @piux2 in https://github.com/gnolang/gno/pull/2471#discussion_r1703064246_

**Comment by zivkovicmilos** [comment]
cc @moul for visibility

**Comment by moul** [comment]
Yes, that's the plan. Every r/sys contract will be either adminless or managed by GovDAO.
